src/NCMenuBar.cc

```diff
@@ -269,44 +279,117 @@ NCMenuBar::wHandleHotkey( wint_t key )
 }
 
 
-void
-NCMenuBar::select_next_menu()
+NCursesEvent
+NCMenuBar::handlePostMenu(const NCursesEvent & event, int selection)
 {
-    std::vector<Menu*>::iterator begin = _menus.begin();
+    NCursesEvent new_event = NCursesEvent::none;
 
-    if (_selected_menu)
+    if ( event == NCursesEvent::button && selection >= 0 )
```

Comment:
`selection` is the index of the currently selected item and probably -1 if nothing is selected?

If that is the case, why not use `selectedIndex` so readers are not confused with an index vs. a pointer (to a YMenuItem or whateve)?